wiki/Glossary/en.md

```diff
@@ -593,7 +594,7 @@ Ouendan has three meanings:
 
 ### Overmapping
 
-Overmapping is when the hit objects or placement of hit objects are placed in a way that makes the difficulty of the map contrast the actual song.
+A term used when the hit objects or placement of hit objects are placed in a way that makes the difficulty of the map contrast the actual song.
 
 This can be done by adding unnecessary jumps, slider speed-ups, or by mapping to an unknown beat in the music consistently (ie. adding notes on blue ticks when the song is only using red ticks), in order to add difficulty.
```

Comment:
```suggestion
This can be done by adding unnecessary jumps, slider speed-ups, or by mapping to an unknown beat in the music consistently (i.e. adding notes on blue ticks when the song is only using red ticks), in order to add difficulty.
```